modules/ibc-hooks/simapp/app.go

```diff
@@ -663,32 +614,31 @@ func NewSimApp(
 
 	// availableCapabilities := strings.Join(AllCapabilities(), ",")
 	wasmDir := filepath.Join(homePath, "wasm")
-	wasmConfig, err := wasm.ReadWasmConfig(appOpts)
+	nodeConfig, err := wasm.ReadNodeConfig(appOpts)
 	if err != nil {
 		panic(fmt.Sprintf("error while reading wasm config: %s", err))
 	}
 
 	// The last arguments can contain custom message handlers, and custom query handlers,
 	// if we want to allow any custom callbacks
-	// availableCapabilities := strings.Join(AllCapabilities(), ",")
```

Comment:
Undo this, it is not related to IBC capability (it is the wasmd capabilities, so good to just keep there)

Reply:
this was a duplicated comment -> 
it exists at a proper place..  let me bring that one here, it uses wasmd function `wasmkeeper.BuiltInCapabilities()`..